### ISSUE TITLE
program.bat file is missing for FW3. 

### ISSUE THREAD

**Issue by ramhen0** [furrtek/PrecIR#17]
Is it going to be same as FW2? Also, I wondering how does the ESL Blaster connects with the app?

**Comment by furrtek** [comment]
Yes it's the same, just replace `ESLBlaster02.bin` with `ESLBlaster02.bin`.
The ESL Blaster connects via USB to the Android app, it's handled as a virtual serial port.

**Comment by ramhen0** [comment]
Thank you for prompt response. I am guessing you meant to say ESLBlaster02.bin with ESLBlaster03.bin?

Also how would it function on “field”? I am guessing the battery can be use to power it and then app on my phone connects via virtual serial port?

I just bought one :)

Sent from my iPhone

On Feb 19, 2024, at 10:45 PM, Furrtek ***@***.***> wrote:

﻿

Yes it's the same, just replace ESLBlaster02.bin with ESLBlaster02.bin.
The ESL Blaster connects via USB to the Android app, it's handled as a virtual serial port.

**Comment by ramhen0** [comment]
After using the app, there are some of my observations.
1) It will help if it was manually added pixel length and width rather than slider. As a requirement, the size is supposed to be multiple of 8, the sliding part is repeated several times before it lands on the multiple of 8 value. Without selecting the multiple of 8, the image is not transferred.
2) I have been able to transfer 120*120 px image. Anything more than that is not being send. Also, the image occupies the left side of the tag. I think max length for px is possibly 120~144? Width would be slight twice the size. (I have a medium sized tag.)
3) If there were instructions on how to use the app itself, it will make the process smooth.

Thank you for the work!
